src/services/product-service.ts

```diff
@@ -9,4 +10,15 @@ export class ProductService {
   static createProduct(newProduct: ICreateProduct) {
     return Api.post<IProduct>(apiUrl().products.create, newProduct);
   }
+  static deleteProduct(id: string) {
+    const product = products.find((p) => p.id === id);
+
+    if (!product) throw new Error("Product not found");
+    products.splice(
+      products.findIndex((p) => p.id === id),
+      1
+    );
```

Comment:
Please just make an api call...  
As we talked the delete should be on the MSW.
```suggestion
```

Reply:
Fixed